manager/allocator/allocator_test.go

```diff
@@ -84,7 +82,7 @@ func TestAllocator(t *testing.T) {
 
 	// Now verify if we get network and tasks updated properly
 	watchNetwork(t, netWatch, false, isValidNetwork)
-	watchTask(t, taskWatch, false, isValidTask)
+	watchTask(t, s, taskWatch, false, isValidTask)
```

Comment:
It seems like I miss something. What this argument propagation is supposed to achieve?